test-integration/task-plan/homework.coffee

```diff
@@ -10,7 +10,7 @@ describe 'Assignment Publishing Tests', ->
   beforeEach ->
     @calendar = new Helpers.Calendar(@)
     @calendarPopup = new Helpers.Calendar.Popup(@)
-    @homework = new Helpers.ReadingBuilder(@)
+    @homework = new Helpers.TaskBuilder(@)
 
```

Comment:
Sorry, I guess I missed updating this reference with the `TaskBuilder` rename :(